css/src/components/navigation.css

```diff
@@ -34,12 +34,23 @@
 }
 
 .Navigation-menu--horizontal {
+  height: var(--spacing-3);
+  padding-right: var(--spacing-l);
+  padding-left: var(--spacing-l);
   border-radius: var(--spacing-2);
-  padding: var(--spacing-m) var(--spacing-l);
   margin: 0 var(--spacing-xs);
 }
 
-.Navigation-menu--active {
+.Navigation-menu--disabled {
+  pointer-events: none;
+}
+
+.Navigation-menu:hover {
+  background-color: var(--secondary-light);
+}
+
+.Navigation-menu--active,
+.Navigation-menu--active:hover {
```

Comment:
Please don't handle hover state for the same. Instead use `pointer-events: none`.